nav2_smac_planner/include/nav2_smac_planner/smac_planner_hybrid.hpp

```diff
@@ -36,6 +36,11 @@
 namespace nav2_smac_planner
 {
 
+/**
+ * Note: All exceptions thrown are handled by the planner server and returned to the action
```

Comment:
This note should be in each of the 3 smac planners when `_a_star->createPath` is called, not here :smile: